cypress.config.js

```diff
@@ -8,12 +9,17 @@ module.exports = defineConfig({
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + `${randomNumber}`,
+            username: faker.person.firstName() + `${randomNumber}`,
             email: 'test' + `${randomNumber}` + '@mail.com',
-            password: 'Password12345!'
+            password: 'Password12345!',
+            country: faker.location.country(),
+            city: faker.location.city(),
```

Comment:
The `city` is generated using `faker.location.city()`. Similar to the country, verify the method name and its return value.